manim/mobject/geometry.py

```diff
@@ -743,6 +742,54 @@ def __init__(self, n=6, **kwargs):
         Polygon.__init__(self, *vertices, **kwargs)
 
 
+class ArcPolygon(VMobject):
+    """
+    The ArcPolygon is what it says, a polygon, but made from arcs.
+    More versatile than the standard Polygon.
+
+    Parameters
+    ----------
+    *arcs : Arc or ArcBetweenPoints
+    
+    Example
+    -------
+    ArcPolygon(arc0,arc1,arc2,arcN,**kwargs)
```

Comment:
so that sphinx displays it properly
```suggestion
    ::
    
        ArcPolygon(arc0,arc1,arc2,arcN,**kwargs)
```